.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+---
+name: 'Feature request'
+about: Making a change to Splink
+title: '<title>'
```

Comment:
Do we want to start using some naming conventions that are dependent on the type of changes being proposed?

`[BUG]`, `[FEAT]`, `[MAINT]`, etc?

These could also be a ticklist item.

Reply:
Have added checklist - feel free to add any more sections that you feel may be relevant